### ISSUE TITLE
Home icon opens a menu, instead of going "home"

### ISSUE THREAD

**Issue by lhilarides** [Vizzuality/mangrove-atlas#1082]
It may be confusing for users that the home icon now opens a menu, instead of navigating "home". 

I'd suggest to use the home icon to navigate "home", and have a separate icon for the menu (hamburger?). I know the GMW logo also functions as a home button, but the top right is not where most people typically look. It's still fine to keep it there too of course!

![image](https://github.com/Vizzuality/mangrove-atlas/assets/3919721/749bcddd-fc28-4ee4-9690-46e80dc690f4)

**Comment by klongleywood** [comment]
@mluena and fyi @lhilarides looks like the home icon was removed completely. Did we want to keep it as an option or is there not enough room?

**Comment by mluena** [comment]
I would say so, adding another icon makes everything look too crowded for me, but let me know what you prefer
<img width="573" alt="Screenshot 2024-04-29 at 08 50 02" src="https://github.com/Vizzuality/mangrove-atlas/assets/33252015/34262020-99bb-4d1b-8b25-e749ac6408c9">

**Comment by klongleywood** [comment]
@lhilarides  do you have a preference here?

**Comment by lhilarides** [comment]
You're right that it becomes too crowded in the menu bar. I gave this some thought, and the functionality we're looking for is to basically "reset the map", which was done by "going home".

I came up with the following solution:
1. Keep the top bar as it is now
2. Add a "Reset map" icon and text to the section in the red rectangle below. This makes both the top-bar and the locations section have 4 items, which should line out nicely. The "reset map" is essentially nothing more than a hyperlink to globalmangrovewatch.org

![image](https://github.com/Vizzuality/mangrove-atlas/assets/3919721/b0d24c7f-611a-4a41-85fa-f8f649a1cda8)
